crutest/src/main.rs

```diff
@@ -1194,40 +1198,31 @@ async fn verify_volume(
     // Steal the write log, so we can share it between threads
     let write_log = std::mem::replace(&mut ri.write_log, WriteLog::new(0));
     let write_log = Arc::new(std::sync::RwLock::new(write_log));
-    let bytes_done = Arc::new(AtomicUsize::new(0));
+    let blocks_done = Arc::new(AtomicUsize::new(0));
 
     let tasks = futures::stream::FuturesUnordered::new();
     for i in 0..NUM_WORKERS {
         let mut block_index = i * IO_SIZE;
         let guest = guest.clone();
         let write_log = write_log.clone();
-        let bytes_done = bytes_done.clone();
+        let blocks_done = blocks_done.clone();
         let total_blocks = ri.total_blocks;
         let block_size = ri.block_size;
         let pb = pb.clone();
         tasks.push(tokio::task::spawn(async move {
             let mut result = Ok(());
+            let mut data = crucible::Buffer::new(0, block_size as usize);
```

Comment:
Does this mean the initial buffer will be filled with 0's?
I believe I had 255 there for some sort of canary, but I no longer
remember what the reason was I put that fence up..

Reply:
No, this means that the initial buffer has a size of 0 blocks; it's always reset before use.

(Admittedly, the reset buffer _will_ be filled with 0s, but after the read, we validate its data against the extremely-not-zero expected pattern, so I don't think we need to guard against it)

Reply:
A blank region will be all zeros, but really we need a few things wrong here to ever have
that situation.